datalad/customremotes/datalad.py

```diff
@@ -93,7 +92,7 @@ def req_CHECKPRESENT(self, key):
         """
         lgr.debug("VERIFYING key %s" % key)
```

Comment:
note: hm, coverage suggests that this piece of code is not covered... that is a bit surprising since `test_basic_scenario_local_url` does hit that code but in the external process of special remote.  I guess at some point we lost proper functioning of merging coverage from multiple reports (i.e. from the main nose process + special remotes which run underneath). And indeed, filed https://github.com/datalad/datalad/issues/4956